vulns/ultralytics/PYSEC-0000-ultralytics.yaml

```diff
@@ -0,0 +1,50 @@
+# TODO: Fill in the details below, and remove any comments prior to committing.
+# See https://ossf.github.io/osv-schema/ for format and field descriptions.
```

Comment:
```suggestion
```

Reply:
Derp, hoisted by my own petard!
That's from my helper tool to generate the advisory, I'll have to add a checker to determine if these remain.